src/the-map.tsx

```diff
@@ -54,8 +61,37 @@ export function TheMap({ selectedRoutes, showUserLocation }: Props) {
                   isAnimatedMarker ? 'animated-marker' : ''
                 }`,
                 html: `<div class="arrow" style="transform: rotate(${p.direction}deg)"></div>`,
-              })}
-            />
+              })}>
+              <Popup>
+                {
+                  // TODO - look up the tracker ID in the vehicles.csv and extract
+                  //        the "accessibility" flag, to decide whether to show
+                  //        the corresponding icon or not
+                  //      - find a way to determine the current route of this vehicle
+                  //        and display it in a popup
+                }
+                {t('label.board')}
+                {':'} {p.board}
+                {/* {vehicles.get(p.board) !=null & '♿'} */}
```

Comment:
@iamandrewluca, I understood how to simply include the icon in there, but there is a slightly more complicated scenario - we have to check

- is this board in the `vehicles` map?
- and if it is, is `accessibility = "yes"`?
- and only then do we render the icon

I was trying to squeeze a few IF statements here, but clearly this doesn't look well. My guess is that perhaps there should be a `Vehicle` structure (not the one from the CSV) with properties, and then we could do that by referring to those properties, like `{p.accessible & '♿'}`.

Any hints about this will be greatly appreciated.

Reply:
When rendering you should use [conditional (ternary) operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) or [Short-circuit evaluation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators#Short-circuit_evaluation)

```jsx
<div>
	{ /* here `accessibility` should be a normalized boolean */ }
	{vehicles.has(p.board) && vehicles.get[p.board].accessibility && (
		rendered content when vehicle exists and has a11y
	)}
</div>
```

Reply:
Done. It looks somewhat cumbersome to me, but if that's the best practice - I can live with it :-)